### PR TITLE
Prevent min/max buttons overlap in panels

### PR DESCRIPTION
**Pull request by javierluraschi**
Prevents...

screen shot 2017-10-06 at 8 16 44 am

to instead...

screen shot 2017-10-06 at 8 16 31 am

### CODE REVIEW

File: src/gwt/src/org/rstudio/studio/client/workbench/ui/WorkbenchTabPanel.java [hunk 1]
```diff
@@ -44,11 +48,29 @@
 {
    public WorkbenchTabPanel(WindowFrame owner, LogicalWindow parentWindow)
    {
+      final int UTILITY_AREA_SIZE = 52;
+      panel_ = new LayoutPanel();
+      
       parentWindow_ = parentWindow;
+
       tabPanel_ = new ModuleTabLayoutPanel(owner);
+      // tabPanel_ = new DocTabLayoutPanel(true, 65, UTILITY_AREA_SIZE);
```
Comment (jmcphers) [inline src/gwt/src/org/rstudio/studio/client/workbench/ui/WorkbenchTabPanel.java]: Do you think we'll need this code in the future? (can we remove it if not, or add an auxiliary comment if so?)

**Comment by jmcphers** [comment]
This problem has existed for a long time and is unfortunately always more obvious when RStudio is displayed on a projector at a high zoom level. Thanks so much for tackling it!

File: src/gwt/src/org/rstudio/studio/client/workbench/ui/WorkbenchTabPanel.java (hunk 1 above)
Comment (javierluraschi) [reply src/gwt/src/org/rstudio/studio/client/workbench/ui/WorkbenchTabPanel.java]: Oops, no that should be removed, fixed this one while my wife was driving us to Portland =P Removed.

Comment (jmcphers) [reply src/gwt/src/org/rstudio/studio/client/workbench/ui/WorkbenchTabPanel.java]: Glad to hear it wasn't while *you* were driving. Safety first!

**Pull request merged**